audioplayer.cpp

```diff
@@ -0,0 +1,95 @@
+#include "audioplayer.h"
+
+
+void AudioPlayer::getFileNames(const string &folder, vector<string>) {
+    vector<string> names;
+    DIR *directory;
+    struct dirent *ent;
```

Comment:
Why do we need struct here? Also, this should be moved near the while loop - nearby it's initialization.

Reply:
Idk. Struct replaced.